main_test.go

```diff
@@ -593,26 +593,26 @@ type proxy struct {
 func setupNewHTTPServer(maxConns uint64, idleTimeout time.Duration) (*Server, error) {
 	s := NewServer(validToken, maxConns, idleTimeout, false, utils.QUIET)
 	var err error
-	ready := make(chan bool)
+	chListenOn := make(chan string)
```

Comment:
Naming is one of the most important things in my opinion. I also stop and think about names a lot...